ocw/lib/gce.py

```diff
@@ -59,7 +60,19 @@ def _delete_resource(self, api_call, resource_name, *_, **kwargs) -> None:
             if GCE.get_error_reason(err) == 'resourceInUseByAnotherResource':
                 self.log_dbg(f"{resource_type.title()} '{resource_name}' can not be deleted because in use")
             elif GCE.get_error_reason(err) == 'badRequest':
-                self.log_err(f"{resource_type.title()} '{resource_name}' can not be deleted because of unknown reason")
+                # These are system generated routes when you create a network. These
+                # will be deleted by the deletion of the network and do not block the
+                # deletion of that network.
+                # There are no properties on the Route struct that indicate a route is a
+                # default one. Typically, the name will contain the word "default" or the
+                # description will contain the word "Default" but a property like Kind
+                # returns "compute#route" for all routes.
+                # All this creating false alarms in log which we want to prevent.
+                # Only way to prevent is mute error
+                if resource_type.title() == "Route" and "The local route cannot be deleted" in str(err):
```

Comment:
Why use `.title()` here?

Reply:
I don't know I was just copy/paste this from your code above :) returning question back to you :)

Reply:
I don't remember XD